src/gui/widgets/template_list_widget.cpp

```diff
@@ -235,8 +229,11 @@ TemplateListWidget::TemplateListWidget(Map* map, MapView* main_view, MapEditorCo
 	adjust_button->setCheckable(true);
 	
 	auto edit_menu = new QMenu(this);
+	georef_action = edit_menu->addAction(tr("Georeferenced"), this, SLOT(changeGeorefClicked()));
```

Comment:
Suggest a more descriptive label for the menu entry. "Use georeferencing" or similar might better express changes associated with checking and unchecking the item than plain "georeferenced"..